### PR TITLE
Improve README example

### PR DESCRIPTION
**Pull request by lostella**
*Description of changes:* Improve simple example in README, in particular
- Show how to load data from a DataFrame
- Explicitly split data into training/test
- Avoid back-and-forth with pandas for plotting reasons (it's in a DataFrame to begin with)

The example is now more verbose, but I think it's clearer (it's not really about it being short, is it?)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Please tag this pr with at least one of these labels to make our release process faster:** BREAKING, new feature, bug fix, other change, dev setup

### CODE REVIEW

**Comment by jaheba** [comment]
The torch version appears to work worse in this example:

Screenshot 2023-01-05 at 14 34 31


Can we reduce the whitespace? The example looks to cluttered for me.

**Comment by lostella** [comment]
I got also better runs, but let's keep the model as it is now maybe. Reduced white space a bit, but not too much.

**Comment by jaheba** [comment]
Does it make sense to split up the imports as well:

```py

# Load data from a CSV file into a PandasDataset
import pandas as pd
from gluonts.dataset.pandas import PandasDataset

df = pd.read_csv(
    "https://raw.githubusercontent.com/AileenNielsen/"
    "TimeSeriesAnalysisWithPython/master/data/AirPassengers.csv",
    index_col=0,
    parse_dates=True
)
dataset = PandasDataset(df, target="#Passengers")

# Split dataset into train/test
from gluonts.dataset.split import split

training_data, test_gen = split(dataset, offset=-36)

# Train a DeepAR model
from gluonts.mx import DeepAREstimator, Trainer

model = DeepAREstimator(
    prediction_length=12, freq="M", trainer=Trainer(epochs=5)
).train(training_data)

# Generate test instances and predictions for them
test_data = test_gen.generate_instances(prediction_length=12, windows=3)
forecasts = list(model.predict(test_data.input))

# Plot predictions
import matplotlib.pyplot as plt

df["#Passengers"].plot(color="black")
for forecast, color in zip(forecasts, ["green", "blue", "purple"]):
    forecast.plot(color=f"tab:{color}")

plt.legend(["True values"], loc="upper left", fontsize="xx-large")
```

**Comment by jaheba** [comment]
Actually, I think that makes it worse!

**Comment by lostella** [comment]
Yeah I think it's fine to have them on top, it's just 5 lines

File: README.md [hunk 1]
```diff
@@ -36,26 +36,40 @@ of data.
...
-from gluonts.dataset.repository.datasets import get_dataset
+from gluonts.dataset.split import split
 from gluonts.mx import DeepAREstimator, Trainer
 
-dataset = get_dataset("airpassengers")
+# Load data from a CSV file into a PandasDataset
 
-deepar = DeepAREstimator(prediction_length=12, freq="M", trainer=Trainer(epochs=5))
-model = deepar.train(dataset.train)
+url = (
```
Comment (jaheba) [inline README.md]: Should we do it in one expression?

```py
# Load data from a CSV file into a PandasDataset
dataset = PandasDataset(
    pd.read_csv(
        "https://raw.githubusercontent.com/AileenNielsen/"
        "TimeSeriesAnalysisWithPython/master/data/AirPassengers.csv",
        index_col=0,
        parse_dates=True,
    ),
    target="#Passengers"
)
```

Comment (lostella) [reply README.md]: Yes, looks good, but I did it in to: constructing the dataframe, wrapping it into a dataset. I think this separation makes sense since it's what one would do interactively (allows to inspect the dataframe)

Comment (lostella) [reply README.md]: Plus, it ends up even shorter :-)

**Review by jaheba — APPROVED** [review]

**Comment by jaheba** [comment]
Do we also want to have this for the current docs?

**Comment by lostella** [comment]
Yes, it works on v0.11 too, so we can backport it
